Mathlib/RingTheory/Polynomial/Basic.lean

```diff
@@ -180,6 +180,71 @@ theorem eval_eq_sum_degreeLTEquiv {n : ℕ} {p : R[X]} (hp : p ∈ degreeLT R n)
   exact (sum_fin _ (by simp_rw [zero_mul, forall_const]) (mem_degreeLT.mp hp)).symm
 #align polynomial.eval_eq_sum_degree_lt_equiv Polynomial.eval_eq_sum_degreeLTEquiv
 
+theorem degreeLT_eq_degreeLE {n : ℕ} : degreeLT R (n + 1) = degreeLE R n := by
+  ext x
+  by_cases x_zero : x = 0
+  · simp_rw [x_zero, Submodule.zero_mem]
+  · rw [mem_degreeLT, mem_degreeLE, ← natDegree_lt_iff_degree_lt (by rwa [ne_eq]),
+      ← natDegree_le_iff_degree_le, Nat.lt_succ]
+
+/-- For every polynomial `p` in the span of a set `s : K[X]`, there exists a polynomial of `p' ∈ s`
+  with higher degree. See also `degree_span'` -/
+theorem degree_span {s : Set R[X]} {p : R[X]} (hs : s.Nonempty) (hp : p ∈ Submodule.span R s) :
```

Comment:
This name is quite low on information. I would prefer something like `degree_le_of_mem_span` or even `exists_degree_le_of_mem_span`. For `degree_span'`, I would append `of_finite` instead of an apostrophe.